.github/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+_Description of changes (if multi-commit, short global summary & context;
+if single-commit, feel free to leave empty)._
+
+---
+
+[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)
+
+Supported PR commands:
+
+Command | Description
+--- | ---
+`extended test - examples` | Live examples tests
+`extended test - storybook` | Storybook screenshot tests
+
+See more [options](https://confluence.intgdc.com/display/~petr.dolejsi/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).
+
+# PR Checklist
+
+- [ ] Verify code changes ([checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
+- [ ] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
+- [ ] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
+- [ ] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
+- [ ] Successful `extended test - examples`
+- [ ] Successful `extended test - storybook`
+- [ ] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)
```

Comment:
How about [ ]  Squash all commit into one?

Reply:
this information is already part of `Check PR owner responsibilities` link mentioned in the template...

Do you feel that its important to you to have it here?